Jenkinsfile.build

```
@@ -1,15 +1,25 @@
-// one job per arch (for now) that just builds "the top thing" (triggered by the meta-update job)
+// any number of jobs per arch that build the specified buildId (triggered by the respective trigger job)
 properties([
-	disableConcurrentBuilds(),
+	// limited by one job per buildId so that the same build cannot run concurrently
+	throttleJobProperty(
+		limitOneJobWithMatchingParams: true,
+		paramsToUseForLimit: 'buildId',
+		throttleEnabled: true,
+		throttleOption: 'project',
+	),
 	disableResume(),
 	durabilityHint('PERFORMANCE_OPTIMIZED'),
 	parameters([
+		string(name: 'identifier', trim: true),
 		string(name: 'buildId', trim: true),
```

Let's put the "required" parameter first and give the `identifier` a description so we remember what it is while triggering the job manually (also doubles as a "load-bearing" code comment :smile:):

```suggestion
		string(name: 'buildId', trim: true),
		string(name: 'identifier', trim: true, description: '(optional) used to set <code>currentBuild.displayName</code> to a meaningful value earlier'),
```